### PR TITLE
Sphinx docs for tutorials

### PR DESCRIPTION
**Pull request by sachinchitta**

### CODE REVIEW

File: pr2_moveit_tutorials/planning/launch/move_group_interface_tutorial.launch [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<launch>
```
Comment (isucan) [inline pr2_moveit_tutorials/planning/launch/move_group_interface_tutorial.launch:1]: Should we also install this file?

**Comment by isucan** [comment]
This looks great Sachin, very cool!

File: pr2_moveit_tutorials/planning/launch/move_group_interface_tutorial.launch (hunk 1 above)
Comment (sachinchitta) [inline pr2_moveit_tutorials/planning/launch/move_group_interface_tutorial.launch:1]: I see an install line for the whole launch directory in CMakeLists.txt - won't that install the whole directory anyway?

**Comment by sachinchitta** [comment]
Don't merge this yet - we have a few more changes coming.

File: pr2_moveit_tutorials/planning/launch/move_group_interface_tutorial.launch (hunk 1 above)
Comment (isucan) [inline pr2_moveit_tutorials/planning/launch/move_group_interface_tutorial.launch:1]: That is fine then. Sorry about that.

**Comment by sachinchitta** [comment]
Ok, this can be merged now.

**Pull request merged**